spec/controllers/video_papers_controller_spec.rb

```diff
@@ -200,11 +194,6 @@
         @shared_paper = FactoryBot.create(:shared_paper, :user => @user2, :video_paper => @paper);
       end
 
-      it "should fail in html" do
-        get :unshare, {:id => @paper.id, :user_id => @user2.id}
-        expect(response.status).to eq 406
-      end
-
```

Comment:
Why is this test removed?

Reply:
Same reason as above.  We never call in it in html.